### DISCUSSION TITLE
bro wtf is this, your hooking into yts.mx for free

### DISCUSSION THREAD

**Discussion by omgbox** [KaustubhPatange/Moviesy#29] [General]
wtf is this shit

![Untitled](https://user-images.githubusercontent.com/11356775/126884613-1ea7962f-c8b6-45b0-ba69-1964fe00757b.png)

**Comment by KaustubhPatange** [comment]
I built the torrent streaming inside the app, I built the cast functionality, I built the torrent downloader do you think I am going to give away all that hardwork for free?

If you have no value for hardwork then immediately uninstall the app I absolutely do not care!
